examples/02_model_collaborative_filtering/als_deep_dive.ipynb

```diff
@@ -2,32 +2,31 @@
  "cells": [
```

Comment:
<p>small detail: the output of this cell is not shown</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1538/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Thanks for pointing that out! Will regenerate it.

Reply:
@miguelgfierro, it turns out the output of this cell is still present, as you can see from [my branch](https://github.com/microsoft/recommenders/blob/laserprec/opt-unit-tests-exp/examples/02_model_collaborative_filtering/als_deep_dive.ipynb). Not sure what ReviewNB is reporting a removal there. Maybe a bug on their end 🤣 .

Reply:
weird